### DISCUSSION TITLE
CSS help for a newbie

### DISCUSSION THREAD

**Discussion by chkrefr** [Athou/commafeed#1476] [Q&A]
Just upgrading commafeed after running for v2.6 for a long time and liking some of the new changes but looking to make some minor adjustments to the UI using the custom CSS option (font size/etc).  

The problem is I'm a complete newbie to CSS, I know nothing about it.  Can anyone point me to some resources that would be useful to get my head around the basics and how to apply them for commafeed?

**Comment by RadithSandeepa** [comment]
Check this [tutorial]( https://www.w3schools.com/Css/)
